packages/amplify-provider-awscloudformation/src/utility-functions.js

```diff
@@ -26,6 +27,49 @@ module.exports = {
 
     return transformGraphQLSchema(context, optionsWithUpdateHandler);
   },
+  newSecret: async (context, options) => {
+    const { description, secret, name, version } = options;
+    const client = await new SecretsManager(context);
+    const response = await client.secretsManager
+      .createSecret({
+        Description: description,
+        Name: name,
+        SecretString: secret,
```

Comment:
What lib or strategy  are we using for secret generation? Just curious about it's randomness.

Reply:
Called from here, to store the GitHub PAT: https://github.com/aws-amplify/amplify-cli/pull/5727/files#diff-b272ed7bbe2a81317d520e4b65b5ce234f34280a801cc44ddf5f7ca77969022cR46